modules/building/pages/accessing-private-images.adoc

```diff
@@ -0,0 +1,66 @@
+= Accessing private image repositories
+
+When the image repository visibility is set to `private` (the default), you need to authenticate before you can pull images. Konflux provides a proxy RBAC image server that allows authorized users to pull private images.
+
+== Getting registry login credentials via UI
+
+To access private images locally:
+
+. Navigate to the *Component details* page for your component
+. In the *Registry login information* section, click the OAuth URL link to open the proxy registry server authentication page
+. Copy the token or the podman login command provided on that page
+. Run the login command in your terminal
+
+After logging in, you can pull images from your private repositories on your local machine.
+
+== Creating service accounts for external systems
+
+If you need to pull container images or OCI artifacts from your Konflux component repositories in external systems outside of Konflux (such as Testing Farm or other CI/CD environments), you can create a Kubernetes service account with the necessary permissions. This allows automated systems to authenticate and pull your private images without requiring interactive login.
+
+=== Prerequisites
+
+You need the following information from the *Component details* page:
+
+* Registry URL
+* Username
```

Comment:
What username is this? User who requested token?

Reply:
See the current UI design, 
<img width="927" height="185" alt="Screenshot From 2025-12-16 20-14-57" src="https://github.com/user-attachments/assets/b1d17452-c2e6-4967-9eb1-3b6e0216070e" />

wlin is the username, and the latter URL is the registry URL. I’ll update the documentation to show `podman login -u <username> <registry_url>/<proxy_host>` for better clarity.